activemq-client/src/main/java/org/apache/activemq/management/SizeStatisticImpl.java

```diff
@@ -87,7 +87,7 @@ public synchronized void setTotalSize(long size) {
     /**
      * @return the maximum size of any step
      */
-    public long getMaxSize() {
+    public synchronized long getMaxSize() {
```

Comment:
For Max + Min Size, I think synchronized should be removed from the setter methods and just make the variables volatile instead.